packages/runtime-core/src/scheduler.ts

```diff
@@ -113,7 +118,9 @@ export function queueJob(job: SchedulerJob): void {
 
 function queueFlush() {
   if (!currentFlushPromise) {
-    currentFlushPromise = resolvedPromise.then(flushJobs)
+    currentFlushPromise = resolvedPromise.then(() => {
```

Comment:
I tested that revert this change and the change of line 50.
tsc works fine.

Reply:
Yes. I just thought we can get rid of the type assertion, I can change it back.

Reply:
Reverted.